ui/component/textareaWithSuggestions/index.js

```diff
@@ -1,27 +1,32 @@
 import { connect } from 'react-redux';
+import { doResolveUris } from 'redux/actions/claims';
+import { getChannelFromClaim } from 'util/claim';
+import { makeSelectClaimForUri } from 'redux/selectors/claims';
 import { MAX_LIVESTREAM_COMMENTS } from 'constants/livestream';
+import { selectChannelMentionData } from 'redux/selectors/livestream';
 import { selectShowMatureContent } from 'redux/selectors/settings';
 import { selectSubscriptionUris } from 'redux/selectors/subscriptions';
 import { withRouter } from 'react-router';
-import { selectCanonicalUrlForUri } from 'redux/selectors/claims';
-import { doResolveUris } from 'redux/actions/claims';
-import { selectChannelMentionData } from 'redux/selectors/livestream';
-import ChannelMentionSuggestions from './view';
+import TextareaWithSuggestions from './view';
 
 const select = (state, props) => {
+  const claim = makeSelectClaimForUri(props.uri)(state);
```

Comment:
Please help to start switching to `selectClaimForUri(state, props.uri)` (note the change in function signature as well).